packages/faustwp-core/src/client.ts

```diff
@@ -104,7 +105,10 @@ export function addApolloState(
 }>['pageProps'] {
   if (pageProps?.props) {
     // eslint-disable-next-line no-param-reassign
-    pageProps.props[APOLLO_STATE_PROP_NAME] = client.cache.extract();
+    pageProps.props = {
+      ...omit(pageProps.props, ['__SEED_NODE__', '__TEMPLATE_QUERY_DATA__']),
```

Comment:
We need to properly pass on the Page `props` property so that they are available to the Page component.

Reply:
I don't think we can omit `__SEED_NODE__` and `__TEMPLATE_QUERY_DATA__` as these are used for previews